watcher.go

```diff
@@ -249,18 +255,29 @@ func (w *Watcher) Buffer(tmplID string) bool {
 	return w.bufferTemplates.Buffer(tmplID)
 }
 
-// Register is used to add dependencies to be monitored by the watcher. It sets
+// Register's one or more Notifiers with the Watcher for future use.
+// Tring to register the same Notifier twice will result in an error and none
```

Comment:
minor nit: I think "Tring" might be "Trying"?

Reply:
fixed